terraform/redis/cluster/provision/main.tf

```diff
@@ -49,7 +49,7 @@ resource "aws_elasticache_replication_group" "redis" {
   description                = format("%s redis", var.instance_name)
   node_type                  = local.node_type
   num_cache_clusters         = var.node_count
-  parameter_group_name       = local.parameter_group_names[var.redis_version]
```

Comment:
I dont know why it was there since it was not created by this team. 
Defining the parameter group name in this case is redundant since the IaaS will select the default for the selected engine - which is the same as what this code was attempting to do.